src/linux/minidump_writer.rs

```diff
@@ -226,11 +246,12 @@ impl MinidumpWriter {
         &mut self,
         buffer: &mut DumpBuf,
         dumper: &mut PtraceDumper,
+        mut soft_errors: ErrorList<WriterError>,
         destination: &mut (impl Write + Seek),
     ) -> Result<()> {
         // A minidump file contains a number of tagged streams. This is the number
         // of streams which we write.
-        let num_writers = 17u32;
+        let num_writers = 19u32;
```

Comment:
Dammit! Good catch! As it turns out, "17 + 1 = 18"!

(This is my stupidity. At one point, I had 2 extra streams in the dump because I was writing both the JSON and the Debug-formatted version to the same file before I switched over to just the JSON.)